conjureup/errors.py

```diff
@@ -0,0 +1,7 @@
+from conjureup.app_config import app
+from conjureup.telemetry import track_exception
+
+
+def handle_exception(exc):
+    track_exception(exc.args[0])
+    return app.ui.show_exception_message(exc)
```

Comment:
handle_exception is replacing a bunch of different handlers that sometimes call `EventLoop.remove_alarms()` - should we be calling it here? If not, why not?

show_exception_message doesn't return anything, so we shouldn't return its return value (it'll be None, but this is unclear and we shouldn't make people go look)